test/functional/apps/visualize/_tsvb_chart.ts

```diff
@@ -27,6 +27,23 @@ export default function({ getService, getPageObjects }: FtrProviderContext) {
   const inspector = getService('inspector');
   const kibanaServer = getService('kibanaServer');
   const testSubjects = getService('testSubjects');
+<<<<<<< HEAD
```

Comment:
@sulemanof, didn't you forget to resolve a conflict?

Reply:
@Avinar-24 seems your problem

Reply:
@rockfield Nice catch. Fixed. Thanks.